transactions/src/main/resources/org/jboss/as/txn/subsystem/LocalDescriptions.properties

```diff
@@ -87,3 +87,5 @@ transactions.commit-markable-resource.jndi-name=JNDi name of this CMR resource
 transactions.commit-markable-resource.xid-location=Name of database table associated to this CMR resource
 transactions.commit-markable-resource.batch-size=Batch size for this CMR resource
 transactions.commit-markable-resource.immediate-cleanup=Immediate cleanup associated to this CMR resource
+
+transactions.stale-transaction-time=The time for which transaction is held in memory after it is being completed.
```

Comment:
only for transactions which contains the remote enlistments (aka. which contains the EJB remote call). This should be emphasized in the description as for any other transaction (aka. the txn with DB) the transaction is removed imediatelly after it's committed.

Reply:
fixed